data-prepper-plugins/opensearch/README.md

```diff
@@ -185,6 +185,9 @@ Besides common metrics in [AbstractSink](https://github.com/opensearch-project/d
 - `documentsSuccess`: measures number of documents successfully sent to ES by bulk requests including retries.
 - `documentsSuccessFirstAttempt`: measures number of documents successfully sent to ES by bulk requests on first attempt.
 - `documentErrors`: measures number of documents failed to be sent by bulk requests.
+- `documentNumberOfRetries`: measures number of times bulk requests are retried.
```

Comment:
Again, the code here is tracking the number of times that there is another bulk request. It does not track documents.

Reply:
Can you rename this from `documentNumberOfRetries` to `bulkRequestNumberOfRetries`?
Same for other bulk request related metrics.

Reply:
`bulkRequestNumberOfRetries` makes sense since it's tracking bulk request.